app/models/proxy_config.rb

```diff
@@ -92,10 +92,10 @@ def update_version
     # Double subquery because mysql needs to create a temporary table.
     # You can't run an UPDATE and subquery from the same table without any temporary one.
 
-    config.update_all("#{self.class.table_name}.version = 1 + (#{Arel.sql max_version.to_sql})")
+    config.update_all("version = 1 + (#{Arel.sql max_version.to_sql})")
 
     # Read the value
-    version = config.connection.select_value(config.select(:version))
+    version = config.connection.select_value(config.select(:version))&.to_i
```

Comment:
This is a hack due to the Rails/Postgres adapter to insist in fetching the value as `String`.

Reply:
No need for the null-type checker.